$OpenBSD: patch-interfaces_time_ui,v 1.1.1.1 2010/09/10 13:51:13 ajacoutot Exp $

XXX Do not display NTP configuration, it requires "services" support.

--- interfaces/time.ui.orig	Thu Sep  9 16:00:31 2010
+++ interfaces/time.ui	Thu Sep  9 16:02:47 2010
@@ -360,7 +360,7 @@
         </child>
         <child>
           <object class="GtkLabel" id="label120">
-            <property name="visible">True</property>
+            <property name="visible">False</property>
             <property name="xalign">0</property>
             <property name="label" translatable="yes">_Configuration:</property>
             <property name="use_underline">True</property>
@@ -374,7 +374,7 @@
         </child>
         <child>
           <object class="GtkComboBox" id="configuration_options">
-            <property name="visible">True</property>
+            <property name="visible">False</property>
           </object>
           <packing>
             <property name="left_attach">1</property>
